Octokit/Clients/IRepositoryCommentsClient.cs

```diff
@@ -19,74 +19,141 @@ public interface IRepositoryCommentsClient
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
         /// <param name="number">The comment id</param>
-        /// <returns></returns>
+        /// <returns>A <see cref="CommitComment"/> for the specified comment id.</returns>
         [SuppressMessage("Microsoft.Naming", "CA1716:IdentifiersShouldNotMatchKeywords", MessageId = "Get",
             Justification = "Method makes a network request")]
         Task<CommitComment> Get(string owner, string name, int number);
 
         /// <summary>
+        /// Gets a single Repository Comment by number.
+        /// </summary>
+        /// <remarks>http://developer.github.com/v3/repos/comments/#get-a-single-commit-comment</remarks>
+        /// <param name="repositoryId">The ID of the repository</param>
+        /// <param name="number">The comment id</param>
+        /// <returns>A <see cref="CommitComment"/> for the specified comment id.</returns>
+        [SuppressMessage("Microsoft.Naming", "CA1716:IdentifiersShouldNotMatchKeywords", MessageId = "Get",
+            Justification = "Method makes a network request")]
+        Task<CommitComment> Get(int repositoryId, int number);
+
+        /// <summary>
         /// Gets Commit Comments for a repository.
         /// </summary>
         /// <remarks>http://developer.github.com/v3/repos/comments/#list-commit-comments-for-a-repository</remarks>
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
-        /// <returns></returns>
+        /// <returns>A <see cref="IReadOnlyPagedCollection{CommitComment}"/> of <see cref="CommitComment"/>s for the specified repository.</returns>
```

Comment:
Weird, that syntax should work [according to this article](https://blogs.msdn.microsoft.com/ansonh/2006/09/11/c-xml-documentation-comments-faq/).

In any case, the return tags are currently worthless, but that's because we've simply been parroting the return type. We don't need to do that because the return type describes itself! Instead, we should use this tag to describe the intent of the returned value. But then again, that's usually already described in the summary. So :thumbsup: to getting rid of these and only use them when there's something particular interesting about the return value we need to describe that isn't described by the type.